Curs/src/views/new-evaluation/new-evaluation.js

```diff
@@ -139,7 +139,7 @@ interviewApp.NewEvaluationPage = function(optionParam = {}) {
     }
    
     return `
-        ${interviewApp.NAV(optionParam.navigationLinks)},
+        ${interviewApp.NAV(optionParam.navLinks)},
```

Comment:
Mkay, I guess. :)

Reply:
I'm not sure I understand. :D

Reply:
Benign change, so benign (and rather useless) comment. :stuck_out_tongue: